dags/openshift_nightlies/tasks/install/rosa/rosa.py

```diff
@@ -19,6 +19,9 @@
 # Defines Tasks for installation of Openshift Clusters
 
 class RosaInstaller(AbstractOpenshiftInstaller):
+    def __init__(self, dag, config: DagConfig, release: OpenshiftRelease):
+        super().__init__(dag, config, release)
+        self.exec_config = self.exec_config = executor.get_default_executor_config(self.dag_config, executor_image="airflow-managed-services")
```

Comment:
is this sentence ok? I think there is an extra self.exec_config

Reply:
yep, good catch. Fixed